### PR TITLE
Fix/padding utility

### PR DESCRIPTION
**Pull request by RV-Adrian**

### CODE REVIEW

**Comment by RV-Adrian** [comment]
@ezequielc @alex-deaconu @fjvallarino 
See above.  Going to tag you directly on PRs here, rather than Trello, going forward. Hopefully that is easier to organize. Thanks!

**Comment by ezequielc** [comment]
@RV-Adrian Code looks good. Pending review on my comment on CH. Thanks!

File: web/partials/storage/storage-modal.html [hunk 1]
```diff
@@ -7,7 +7,7 @@ <h3 id="title" class="modal-title" translate="storage-client.select-{{storageFac
   </div>
 
   <div class="modal-body storage-full" stop-event="touchend">
-    <div class="padding-large-horizontal padding-large-bottom">
+    <div class="u_padding-lg-horizontal">
```
Comment (alex-deaconu) [inline web/partials/storage/storage-modal.html]: @RV-Adrian I see the `padding-large-bottom` class is removed. Is that intentional? Thanks.

Comment (RV-Adrian) [inline web/partials/storage/storage-modal.html]: @alex-deaconu Yes, strangely that class doesn't exist in Common Style. May have been deprecated in the past.

Comment (alex-deaconu) [inline web/partials/storage/storage-modal.html]: @RV-Adrian sounds good. Thanks for the clarification.

**Comment by RV-Adrian** [comment]
@ezequielc @alex-deaconu @fjvallarino 
I've updated the tags and staged at https://apps-stage-3.risevision.com/

This is an organization of CS, and just has new class names in Apps. Thanks!

**Comment by alex-deaconu** [comment]
@RV-Adrian looks good, didn't find any issues 👍 . Thanks!

**Pull request merged**